### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by Kume001** [attocash/node#215] [Ideas]
1. Would be nice to have official darkmode theme
2. and also the explorer could be better having a standalone page.
3. Footer seem to be occupying a lot of space on the page .
Glad to be a part of this project. Future is bright. 

atto://aatdnamuvnzns32pp4cgvfjnw5khdx7j5tajypqvj3u4qa5i2peyh27knhtbu

**Comment by rotilho** [comment]
Hey @Kume001!

Thank you so much for the feedback and for being part of the project

Dark mode is absolutely on the roadmap, though it’s planned for a later stage. It’s something we’re excited about too!

For the explorer, could you share more about why you feel it should be standalone? Personally, I like having everything integrated into a single site, but I’d love to understand your perspective better.

As for the footer… yeah, it was huge before and has been trimmed down quite a bit, but I agree it could still use some work. I’ll take another look at it.

Thanks again for the great suggestions!
